app/screens/global_drafts/components/global_drafts_list/global_drafts_list.tsx

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {FlatList} from '@stream-io/flat-list-mvcp';
+import React, {useCallback, useEffect, useState} from 'react';
+import {InteractionManager, StyleSheet, View, type LayoutChangeEvent, type ListRenderItemInfo} from 'react-native';
+import Animated from 'react-native-reanimated';
+import Tooltip from 'react-native-walkthrough-tooltip';
+
+import {storeDraftsTutorial} from '@actions/app/global';
+import {INITIAL_BATCH_TO_RENDER, SCROLL_POSITION_CONFIG} from '@components/post_list/config';
+import {Screens} from '@constants';
+import useAndroidHardwareBackHandler from '@hooks/android_back_handler';
+import {popTopScreen} from '@screens/navigation';
+
+import DraftEmptyComponent from '../draft_empty_component';
+
+import DraftSwipeActions from './DraftSwipeActions';
+import DraftTooltip from './tooltip';
+
+import type DraftModel from '@typings/database/models/servers/draft';
+
+type Props = {
+    allDrafts: DraftModel[];
+    location: string;
+    tutorialWatched: boolean;
+}
+
+const styles = StyleSheet.create({
+    container: {
+        flex: 1,
+    },
+    empty: {
+        alignItems: 'center',
+        flexGrow: 1,
+        justifyContent: 'center',
+    },
+    tooltipStyle: {
+        shadowColor: '#000',
+        shadowOffset: {width: 0, height: 2},
+        shadowRadius: 2,
+        shadowOpacity: 0.16,
+    },
+    swippeableContainer: {
+        width: '100%',
+    },
+    tooltipContentStyle: {
+        borderRadius: 8,
+        width: 247,
+        padding: 16,
+        height: 160,
+    },
+});
+
+const AnimatedFlatList = Animated.createAnimatedComponent(FlatList);
```

Comment:
Should we consider using Animated.Flatlist since createAnimatedComponent(FlatList) is deprecated?

 @deprecated Please use `Animated.FlatList` component instead of calling `Animated.createAnimatedComponent(FlatList)` manually.

Reply:
Done.